client/components/PhotoGallery.jsx

```diff
@@ -1,16 +1,33 @@
 import React from 'react';
+import Photo from './Photo.jsx';
+import withModal from './withModal.jsx';
+
 
 
 class PhotoGallery extends React.Component {
-	constructor(){
-		super();
-	}
-
-	render(){
-		return(
-          <img src="./images/img1.jpg"/>
-		);
-	}
+  render() {
```

Comment:
Since this class only has a render method, It can just be a function which does the work from lines 9-22, and returns what you already return on lines 26-29. In short, this class declaration can be turned into a function declaration, and the line 8 and 30 can be removed.

Reply:
@IsaacLeeWebDev I will get rid of this component, i think i will work on refactoring for components